meson.build

```diff
@@ -0,0 +1,65 @@
+project('eos-metrics',
+  'c',
+  version: '0.5.0',
+  license: 'LGPL2+',
+  meson_version: '>= 0.62'
```

Comment:
Endless OS master has 0.61.1.

(I'll push a commit to add the build-dependency on meson, which is why the build is currently failing, but it will still fail for this reason.)

Reply:
I suspect we don't actually depend on 0.62, I think 0.60 will be fine. This was copy-pasted from other projects.

Reply:
Great, please feel free to adopt my patch. Actually the Jenkins build is now failing because it passes some parameters to `configure`. I tried to fix that in https://github.com/endlessm/eos-build/pull/1435 (private repo) and discovered that the CI that updates the CI is broken; a fix for that is awaiting review on https://github.com/endlessm/eos-build/pull/1436. It's never simple!